lib/routes/account.js

```diff
@@ -918,32 +919,31 @@ module.exports = (
               tokenVerificationId: tokenVerificationId
             })
 
-            return getGeoData(ip)
-              .then(
-                function (geoData) {
-                  return mailer.sendVerifyLoginEmail(
-                    [],
-                    emailRecord,
-                    {
-                      acceptLanguage: request.app.acceptLanguage,
-                      code: tokenVerificationId,
-                      flowId: flowId,
-                      flowBeginTime: flowBeginTime,
-                      ip: ip,
-                      location: geoData.location,
-                      redirectTo: redirectTo,
-                      resume: resume,
-                      service: service,
-                      timeZone: geoData.timeZone,
-                      uaBrowser: sessionToken.uaBrowser,
-                      uaBrowserVersion: sessionToken.uaBrowserVersion,
-                      uaOS: sessionToken.uaOS,
-                      uaOSVersion: sessionToken.uaOSVersion,
-                      uaDeviceType: sessionToken.uaDeviceType
-                    }
-                  )
-                }
-              )
+            const promises = [getGeoData(ip), features.isSecondaryEmailEnabled() ? db.accountEmails(sessionToken.uid) : P.resolve([])]
+            return P.all(promises)
+              .spread((geoData, emails) => {
+                return mailer.sendVerifyLoginEmail(
+                  emails,
+                  sessionToken,
```

Comment:
can this be `emailRecord` like https://github.com/mozilla/fxa-auth-server/pull/1768/files#diff-ab87c506e682e952973178a41792df12R887 ?